src/scripts/visualizations/line.js

```diff
@@ -139,7 +138,7 @@
         }
 
         function renderMarkers() {
-            var animationDelay = options.line.marker.animationDelay;
+            var animationDelay = (typeof options.line.marker.animationDelay === 'undefined') ? duration : options.line.marker.animationDelay;
```

Comment:
u can just check `options.line.marker.animationDelay === undefined` for conciseness